freesound/settings.py

```diff
@@ -281,9 +281,12 @@
 
 SESSION_COOKIE_DOMAIN = None # leave this until you know what you are doing
 
-# Sound analysis properties
+# Sound analysis/processing worker properties
+WORKER_TIMEOUT = 5 * 60  # General timeout for processing/analysis workers to avoid to hang forever
```

Comment:
small change, it'd be nice to see comments before the line that they affect. For this one, indicate that the time is in seconds

Reply:
Addressed in 5b9d860ce7beaed0ffa21f493d76239c47905013